### DISCUSSION TITLE
Home security

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#348] [General]
What kind of home security do you use?
Cameras, alarm system etc?
What company are you with?

**Comment by Bibett** [comment]
I plan to install cameras but don't know which ones are the best. Any recommendations?

**Comment by hernessteve** [comment]
Both actually. Did you hear about vivint? They offer security and protection services in my area and i must admit they are pretty good. We are with them for about two years now and never had any troubles. Just check some reviews about them [](https://vivint.pissedconsumer.com/review.html)https://vivint.pissedconsumer.com/review.html. Hope you will find what you are looking for.
